### DISCUSSION TITLE
Is there a way to pick only month and year ?

### DISCUSSION THREAD

**Discussion by dorapax-lab** [mmazzarolo/react-native-modal-datetime-picker#619] [Q&A] [answered]
Hi,

Is there a way to pick only month and year ?

Thanks.

**Comment by mmazzarolo** [comment, marked as answer]
[Unless the picker we're using under-the-hood allows doing that](https://github.com/react-native-datetimepicker/datetimepicker) (I don't think it does), nope :/
